HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc

```diff
@@ -0,0 +1,282 @@
+// -*- C++ -*-
```

Comment:
HLTrigger/Egamma already defines only plugins (in the src directory), you should not use a new 'plugins' directory, simply put the .cc file under /src .